doc/MAGIC.md

```diff
@@ -0,0 +1,107 @@
+	"max_dot": 2,
+	"dot_increment": 0.1,
+	"min_duration": 0,                                        // duration of spell effect (if the spell has a special effect)
+	"max_duration": 1000,
+	"duration_increment": 4,
+	"min_pierce": 0,                                          // how much of the spell pierces armor (currently not implemented)
+	"max_pierce": 1,
+	"pierce_increment": 0.1
+  }
+```
+Most of the default values for the above are either 0 or "NONE", so you may leave out most of the values if they do not pertain to your spell.
+
+When deciding values for some of these, it is important to note that some of the formulae are not linear.
+For example, this is the formula for spell failure chance: 
+
+```( ( ( ( spell_level - spell_difficulty ) * 2 + intelligence + spellcraft_skill ) - 30 ) / 30 ) ^ 2```
+
+Meaning a spell with difficulty 0 cast by a player with 8 intelligence, 0 spellcraft, and level 0 in the spell will have a 53% spell failure chance.
+On the other hand, a player with 12 intelligence, 6 spellcraft, and level 6 in the same spell will have a 0% spell failure chance.
+
+However, experience gain is a little more complicated to calculate.  The formula for how much experience you need to get to a level is below:
+
+```e ^ ( ( level + 62.5 ) * 0.146661 ) ) - 6200```
+
+#### Currently Implemented Effects and special rules
+
+* "pain_split" - makes all of your limbs' damage even out.
+
+* "move_earth" - "digs" at the target location. some terrain is not diggable this way.
+
+* "target_attack" - deals damage to a target (ignores walls).  If "effect_str" is included, it will add that effect (defined elsewhere in json) to the targets if able, to the body parts defined in effected_body_parts.
+Any aoe will manifest as a circular area centered on the target, and will only deal damage to valid_targets. (aoe does not ignore walls)
+
+* "projectile_attack" - similar to target_attack, except the projectile you shoot will stop short at impassable terrain.  If "effect_str" is included, it will add that effect (defined elsewhere in json) to the targets if able, to the body parts defined in effected_body_parts.
+
+* "cone_attack" - fires a cone toward the target up to your range.  The arc of the cone in degrees is aoe.  Stops at walls.  If "effect_str" is included, it will add that effect (defined elsewhere in json) to the targets if able, to the body parts defined in effected_body_parts.
+
+* "line_attack" - fires a line with width aoe toward the target, being blocked by walls on the way.  If "effect_str" is included, it will add that effect (defined elsewhere in json) to the targets if able, to the body parts defined in effected_body_parts.
+
+* "spawn_item" - spawns an item that will disappear at the end of its duration.  Default duration is 0.
+
+* "teleport_random" - teleports the player randomly range spaces with aoe variation
+
+##### For Spells that have an attack type, these are the available damage types:
+"fire"
```

Comment:
These should be a list too, they currently all render on one line: https://github.com/CleverRaven/Cataclysm-DDA/blob/75c27d30dcf596ef725b8c6a21bdca52e0526a59/doc/MAGIC.md